index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const { getAllMovies, getMovieByTitle, addNewMovie } = require('./controllers/movies')
+
+const app = express()
+
+app.get('/movies', getAllMovies)
+app.get('/movies/:search', getMovieByTitle)
```

Comment:
This route is to search by titles and directors be descriptive when naming routes. For example getMovieByTitleOrDirector